### PR TITLE
Infra: Add db import script

### PR DESCRIPTION
**Pull request by minotogna**
@sorja @yaguzmang 
Add db import script + added intellij/webstorm run configuration .

image

It's basically a command that runs the following command:
```shell
(cd src/tools/db/importBackup && ./exec.sh) && ts-node src/tools/generateCache/index.ts
```

@yaguzmang feel free to add the vscode version if you need it.

### CODE REVIEW

**Comment by sorja** [comment]
to avoid `../../../` you can also cd in subshell using parenthesis

`(cd src/tools/db/importBackup && ./exec.sh) && ts-node src/tools/generateCache/index.ts`

File: src/tools/db/importBackup/_pgRestore.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+dropdb frap-dev --user frap
```
Comment (sorja) [inline src/tools/db/importBackup/_pgRestore.sh]: suggest avoid `-f` for `force` to close open connections (eg. intellij etc might have open connections)

Comment (minotogna) [reply src/tools/db/importBackup/_pgRestore.sh]: sure . pushed

**Comment by minotogna** [comment]
> to avoid `../../../` you can also cd in subshell using parenthesis
> 
> `(cd src/tools/db/importBackup && ./exec.sh) && ts-node src/tools/generateCache/index.ts`

nice 👍 
@sorja @yaguzmang  I have also updated the wiki: https://github.com/openforis/fra-platform/wiki/Developer-Commands:-Database

**Review by sorja — DISMISSED** [review]

File: .run/db_backup-import.run.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="db:backup-import" type="ShConfigurationType">
+    <option name="SCRIPT_TEXT" value="(cd src/tools/db/importBackup &amp;&amp; ./exec.sh) &amp;&amp; ts-node src/tools/generateCache/index.ts" />
+    <option name="INDEPENDENT_SCRIPT_PATH" value="true" />
+    <option name="SCRIPT_PATH" value="cd src/tools/db/importBackup &amp;&amp; ./exec.sh &amp;&amp; cd ../../../../ &amp;&amp; ts-node src/tools/generateCache/index.ts" />
```
Comment (sorja) [inline .run/db_backup-import.run.xml]: Should this be updated?

Comment (minotogna) [reply .run/db_backup-import.run.xml]: I pushed the change https://github.com/openforis/fra-platform/pull/4004/commits/3640af66dad845d20d629ad78bea64a079f41e92

Comment (sorja) [reply .run/db_backup-import.run.xml]: unchanged in `SCRIPT_PATH` but not sure if related

Comment (minotogna) [reply .run/db_backup-import.run.xml]: removed script_path 🙃

**Review by sorja — APPROVED** [review]

**Pull request merged**